documentation/modules/con-securing-client-authorization.adoc

```diff
@@ -36,7 +36,9 @@ For more information on configuring super user access to brokers, see xref:con-s
 
 == User quotas
 
-You can configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed access to Kafka brokers based on a byte threshold or a time limit of CPU utilization.
+You can configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed access to Kafka brokers.
+You can set size-based network usage and time-based CPU utilization thresholds.
+You can also control the rate at which requests to change partitions are accepted for user requests.
```

Comment:
Known as partition mutation quotas?

Reply:
Not sure we need it here. But changed the callout to: _Number of concurrent partition mutations (creation and deletion operations) allowed per second_

Reply:
Changed to: _You can also add a partition mutation quota to control the rate at which requests to change partitions are accepted for user requests._
Updated callout too: _Number of concurrent partition creation and deletion operations (mutations) allowed per second_